guides/common/modules/con_content-views-in-project.adoc

```diff
@@ -19,32 +19,64 @@ However, the repositories for the content views still exist and you can keep man
 A _Default Organization View_ is an application-controlled content view for all content that is synchronized to {Project}.
 With the Default Organization View, you can register a host to {Project} and access content without manipulating content views and lifecycle environments.
```

Comment:
This sentence contradicts my finding that Foreman/Katello only shows synchronized Kickstart repositories, but no further content. That's why my change is non-Satellite only.

Can you a) unportect any RH content in Foreman/Katello and b) consume this unprotected content on RHEL hosts without subscription-manager? If yes and yes, is there a use case/do you want to have this mentioned explicitely? If so, I'd rework my last commit.

cc @ianballou

Reply:
OK, I conclude that a) you cannot unprotect RH content and b) there's really no need to do so due to subscription-manager. -> add docs for Katello/orcharhino only.

Reply:
> a) You cannot unprotect Red Hat content.

But kickstart repos are publicly available because Anaconda needs to boot from it without having any credentials.